javascript_nodejs/14.nlp-with-dispatch/dialogs/homeAutomation/index.js

```diff
@@ -36,7 +36,7 @@ class homeAutomation {
         // home automation state
         this.state = new HomeAutomationState(convoState, userState);
         
-        // add recogizers
+        // add recognizers
```

Comment:
Should also be `"Add the LuisRecognizer."` for this line and the other changes below, unless there are actually multiple recognizers being registered.